rfc/2019-11-30-recommand-tidb-configure.md

```diff
@@ -0,0 +1,32 @@
+# Proposal: Official recommended topologies and configurations to use TiDB
+
+- Author(s): ET-Team
+- Last updated:  Nov 30, 2019
+
+## Abstract
+
+When users need to use TiDB as their database, they often face what machine configuration and topology to choose. At present, we do not give an official answer, which may cause users to be troubled, what kind of The configuration is to meet the current needs. This proposal is mainly to solve the user's selection problem.
+
+## Background
+
+Recommending configurations to users is a relatively standard practice for major public cloud vendors. When AWS recommends EC2 to users, it will give some standard instance types. As an open source database, according to the user's characteristic needs give different topologies  and database configuration and machine configuration will greatly help users to choose and better use TiDB
```

Comment:
```suggestion
Recommending configurations to users is a relatively standard practice for major public cloud vendors. When AWS recommends EC2 to users, it will give some standard instance types. As an open source database, according to the user's characteristic needs give different topologies and database configuration and machine configuration will greatly help users to choose and better use TiDB
```